starboard/nplb/BUILD.gn

```diff
@@ -13,9 +13,15 @@
 # limitations under the License.
 
 import("//starboard/build/config/os_definitions.gni")
+import("//starboard/contrib/cast/cast.gni")
```

Comment:
Is it possible for us to split the PR into two:
1. For changes specific to cast, like the CastCodecsAreSupported(), keep them into a test target inside `//starboard/contrib/cast/`, and ensure that relevant partners understand that they have to pass them.
2. For changes that are general for all platforms (like refining `VideoDmpReader::audio_mime_type()`), merge them into main line starboard (not contrib).

Reply:
Makes sense; done. I'll send out a separate PR for the changes under //starboard/contrib/cast